packages/stark-ui/src/modules/app-sidebar/components/app-sidebar.component.ts

```diff
@@ -128,6 +143,10 @@ export class StarkAppSidebarComponent extends AbstractStarkUiComponent implement
 		this.breakpointObserver.observe([this.mediaQueryGtLg]).subscribe((state: BreakpointState) => {
 			this.onObserveBreakpoints(state);
 		});
+
+		this.routingService.addTransitionHook(StarkRoutingTransitionHook.ON_SUCCESS, {}, () => {
```

Comment:
We should deregister the transition hook in the `ngOnDestroy` to prevent memory leaks.

The `routingService.addTransitionHook()` method returns the function that should be called to deregister the hook ;)